plugin/build.gradle

```diff
@@ -31,13 +31,16 @@ repositories {
 
 dependencies {
     implementation gradleApi()
-    api "org.jetbrains.kotlin:kotlin-stdlib-jdk7:1.3.50"
+    api 'org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.5.10'
```

Comment:
this may be an issiue when we add it to our project. I'm not sure at the moment. Lets try.